tests/builds/cjs.js

```diff
@@ -0,0 +1,9 @@
+var assert = require('assert');
```

Comment:
Importing spec.js, and running it again here and in the esm test would be perfect.
It might be a bit much for this PR though.
If it's too much work, I'll try to do it myself later. :)

Reply:
I think this requires some tasks a bit. so I'd like to possible to avoid in this PR 🙇

Reply:
Sure thing, sounds good to me :)